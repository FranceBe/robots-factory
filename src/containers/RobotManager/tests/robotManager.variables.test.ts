import {
  requirementsForAFoobar,
  requirementsForARobot,
  tooltipTexts,
} from 'containers/RobotManager/robotManager.variables'

describe('RobotManager variables', () => {
  describe('requirementsForAFoobar', () => {
    it('should match snapshot', () => {
      expect(requirementsForAFoobar).toMatchSnapshot()
    })
  })
  describe('requirementsForARobot', () => {
    it('should match snapshot', () => {
      expect(requirementsForARobot).toMatchSnapshot()
    })
  })
  describe('tooltipTexts', () => {
    it('should match snapshot', () => {
      expect(tooltipTexts).toMatchSnapshot()
    })
  })
})
