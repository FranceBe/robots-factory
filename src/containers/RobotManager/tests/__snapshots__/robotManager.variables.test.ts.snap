// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RobotManager variables requirementsForAFoobar should match snapshot 1`] = `
Object {
  "bar": 1,
  "foo": 1,
}
`;

exports[`RobotManager variables requirementsForARobot should match snapshot 1`] = `
Object {
  "foo": 6,
  "foobar": 3,
}
`;

exports[`RobotManager variables tooltipTexts should match snapshot 1`] = `
Object {
  "foobar": "<p>Requiert <span class=\\"foobar-foo\\">1 Foo</span> et <span class=\\"foobar-bar\\">1 Bar</span>.</p>",
  "robot": "<p>Requiert <span class=\\"robot-foobar\\">3 Foobar</span>  et <span class=\\"robot-foo\\">6 Foo</span>.</p>",
}
`;
